index.ts

```diff
@@ -676,23 +740,64 @@ export default class FreeSound {
     return this.makeRequest(this.makeUri(this.uris.upload), 'POST', formData);
   }
 
+  /**
+   * Describe a previously uploaded file that does not have a description.
+   * Note: after a sound receives a description, the team of Freesound moderators
+   * still needs to process and moderate it, so it may not yet appear in Freesound.
+   * A list of sounds uploaded and described by the user, but still
+   * pending processing and moderation, is viewable with
+   * the getPendingSounds() method. This method requires OAuth2 authentication. See
+   * https://freesound.org/docs/api/resources_apiv2.html#describe-sound-oauth2-required
+   * for more information.
+   * @param description a description for an uploaded sound
+   */
   describe(description: { description: string }) {
     this.checkOauth();
     const formData = this.makeFormData(description);
     return this.makeRequest(this.makeUri(this.uris.upload), 'POST', formData);
   }
 
+  /**
+   * Retrieve a list of audio files uploaded by
+   * the Freesound user logged in using OAuth2 that
+   * do not have a description, or have not been
+   * processed or moderated. In Freesound, sounds
+   * need descriptions after their upload. Then,
+   * sounds are automatically processed, and,
+   * finally, a team of human moderators either
+   * accepts or rejects the upload. This method keeps
+   * track of the status of these uploads and
+   * requires OAuth2 authentication. See
+   * https://freesound.org/docs/api/resources_apiv2.html#pending-uploads-oauth2-required
+   * for more information.
+   * ```typescript
+   * const result = await freeSound.getPendingSounds()
+   * ```
+   */
   getPendingSounds() {
     this.checkOauth();
     return this.makeRequest(this.makeUri(this.uris.pending));
   }
 
-  // user resources
+  /**
+   * Return basic information about the user that is logged in via OAuth2.
+   * This application can use it to identify which Freesound user has logged in.
+   * ```typescript
+   * const result = await freeSound.me();
+   * ```
+   */
   me() {
     this.checkOauth();
     return this.makeRequest(this.makeUri(this.uris.me));
   }
 
+  /**
```

Comment:
Missing method summary

Reply:
Ah, I didn't add method summaries for `getLoginURL(): string` and `setClientSecrets(id: string, secret: string)` because I didn't see any [search results for "getLoginURL" in the Freesound API](https://freesound.org/docs/api/search.html?q=getLoginURL), or for ["setClientSecrets"](https://freesound.org/docs/api/search.html?q=setClientSecrets).

We can add our own descriptions for these methods. It's straightforward to describe what "setClientSecrets" does; for "getLoginURL", I might say something like, "Navigate to Freesound for user login."